Token_Contracts/test/humanStandardToken.js

```diff
@@ -130,6 +130,21 @@ contract("HumanStandardToken", function(accounts) {
         }).catch(done);
```

Comment:
In a next PR, please remove this and all other `catch(done)`.  `done` should only be called once per test.